### PR TITLE
(PC-32684) feat(Achievements): added real data

### PR DESCRIPTION
**Pull request by cgerrard-pass**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-32684

## Flakiness

If I had to re-run tests in the CI due to flakiness, I add the incident [on Notion][2]

## Checklist

I have:

- [x] Made sure my feature is working on web.
- [x] Made sure my feature is working on mobile (depending on relevance : real or virtual devices)
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [x] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change

### CODE REVIEW

File: src/features/profile/pages/Achievements/AchievementData.ts [hunk 1]
```diff
@@ -1,239 +1,125 @@
...
+  FIRST_MOVIE_BOOKING = 'FIRST_MOVIE_BOOKING',
+  FIRST_BOOK_BOOKING = 'FIRST_BOOK_BOOKING',
+  FIRST_RECORDED_MUSIC_BOOKING = 'FIRST_RECORDED_MUSIC_BOOKING',
+  FIRST_SHOW_BOOKING = 'FIRST_SHOW_BOOKING',
+  FIRST_MUSEUM_BOOKING = 'FIRST_MUSEUM_BOOKING',
+  FIRST_LIVE_MUSIC_BOOKING = 'FIRST_LIVE_MUSIC_BOOKING',
+  FIRST_NEWS_BOOKING = 'FIRST_NEWS_BOOKING',
+  FIRST_INSTRUMENT_BOOKING = 'FIRST_INSTRUMENT_BOOKING',
+  FIRST_ART_LESSON_BOOKING = 'FIRST_ART_LESSON_BOOKING',
+  TEST = 'TEST',
```
Comment (lbeneston-pass) [inline src/features/profile/pages/Achievements/AchievementData.ts:24]: C'est un oubli le TEST ?

File: src/features/profile/pages/Achievements/AchievementData.ts [hunk 2]
```diff
@@ -1,239 +1,125 @@
...
-  {
-    id: 'THIRD_WATCH_MOVIE',
-    name: 'Premier festival',
-    description: 'Tu as participé à ton premier festival\u00a0!',
-    category: 'Découvre le catalogue',
-    icon: 'Festival',
-  },
+export enum AchievementCategory {
+  FIRST_BOOKINGS = 'FIRST_BOOKINGS',
+  TEST = 'TEST',
```
Comment (lbeneston-pass) [inline src/features/profile/pages/Achievements/AchievementData.ts:29]: Idem

File: src/features/profile/pages/Achievements/useAchievementDetails.ts [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-  completedAchievements,
+  mockAchievements,
+  mockCompletedAchievements,
 } from 'features/profile/pages/Achievements/AchievementData'
 
 export const useAchievementDetails = (id: string) => {
-  const achievement = achievements.find((achievement) => achievement.id === id)
-  const completedAchievement = completedAchievements.find((u) => u.id === id)
+  const achievement = mockAchievements.find((achievement) => achievement.id === id)
+  const completedAchievement = mockCompletedAchievements.find((u) => u.id === id)
```
Comment (lbeneston-pass) [inline src/features/profile/pages/Achievements/useAchievementDetails.ts:8]: "u" c'est pas très explicite

File: src/features/profile/pages/Achievements/useAchievements.native.test.ts [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+  firstMovieBooking,
+  userCompletedArtLessonBooking,
+  userCompletedBookBooking,
+  userCompletedMovieBooking,
+} from 'features/profile/pages/Achievements/AchievementData'
+import { useAchievements } from 'features/profile/pages/Achievements/useAchievements'
+import { renderHook } from 'tests/utils'
+
+const testAchievement = {
+  id: AchievementId.TEST,
```
Comment (lbeneston-pass) [inline src/features/profile/pages/Achievements/useAchievements.native.test.ts:16]: Ah je viens de comprendre. Je pense que pour les tests tu peux utiliser un vrai achievementId

File: src/features/profile/pages/Achievements/useAchievements.native.test.ts [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+
+        expect(badges).toEqual([
+          expect.objectContaining({
+            category: AchievementCategory.FIRST_BOOKINGS,
+            progress: 0.75,
+          }),
+        ])
+      })
+
+      describe('text', () => {
```
Comment (lbeneston-pass) [inline src/features/profile/pages/Achievements/useAchievements.native.test.ts:258]: Un oubli ?

Comment (cgerrard-pass) [reply src/features/profile/pages/Achievements/useAchievements.native.test.ts:275]: Quel est l'oubli?

Comment (lbeneston-pass) [reply src/features/profile/pages/Achievements/useAchievements.native.test.ts:276]: `describe('test', () => { 
`
test ?

Comment (lbeneston-pass) [reply src/features/profile/pages/Achievements/useAchievements.native.test.ts:276]: Ok je vois, non pas d'idée de titre plus descriptif

**Review by lbeneston-pass — APPROVED** [review]

**Pull request merged**